tests/app/main/views/test_jobs.py

```diff
@@ -230,7 +230,7 @@ def test_should_show_job_with_sending_limit_exceeded_status(
     )
 
     assert normalize_spaces(page.select('main p')[1].text) == (
-        "Notify cannot send these messages because you have reached your daily limit. You can only send 1,000 messages per day."  # noqa
+        "Notify cannot send these messages because you have reached a limit. You can only send 1,000 messages per day and 250,000 messages in total."  # noqa
```

Comment:
@stvnrlly is there a reason to say 1000 per day or is this essentially a placeholder until we determine this number?

Reply:
is `current_service.message_limit` 1000?

Reply:
1000 as a daily limit was chosen randomly for the test.  In the code itself, that limit is 5000.

Reply:
ok, we can change that in the future if need be.